tests/integration_test.go

```diff
@@ -0,0 +1,90 @@
+package tests
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"testing"
+
+	skynet "github.com/NebulousLabs/go-skynet"
+	"gopkg.in/h2non/gock.v1"
+)
+
+// TestUploadAndDownloadFile tests uploading and downloading a single file.
+func TestUploadAndDownloadFile(t *testing.T) {
+	defer gock.Off() // Flush pending mocks after test execution
+
+	const srcFile = "../testdata/file1.txt"
```

Comment:
Looks like this code is already duplicated between these two test and I imagine it will be used for any future testing. Can we extract it into a helper similar to the `siatest TestGroups`?